.github/workflows/deploy.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy
+
+run-name: Site updated to ${{ github.ref }}
+
+on:
+  workflow_dispatch: {}
+  workflow_run:
+    workflows: [CI]
+    types: [completed]
+    branches: [main]
+
+jobs: 
+  build-and-deploy:
+    name: Publish to GH pages
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/main' }}
+    permissions: 
+      contents: read
+      pages: write
+      id-token: write
+    steps: 
+      - uses: actions/checkout@v4
+      - uses: actions/setup-ruby@v1
+      - name: Install dependencies
+        run: |
+          gem install bundler
+          bundle install
+      - name: Setup Pages
+        uses: actions/configure-pages@v5
+      - name: Build
+        uses: actions/jekyll-build-pages@v1
```

Comment:
Is this the right action to be using here?  We don't use Jekyll (as far as I can see) and [normally](https://github.com/alphagov/govuk-content-api-docs/blob/b4c34c769c8f4ac6bd01b86ba4af07a261eb6492/Makefile#L8C2-L8C24) we'd run `rake build` (as they're doing in the [equivalent action](https://github.com/alphagov/govuk-developer-docs/blob/082c39abb66fc544dac29c61e0d94109ba629805/.github/workflows/deploy.yml#L47) in the dev docs).

Reply:
Snap